document/src/vespa/document/update/documentupdate.h

```diff
@@ -101,6 +101,9 @@ class DocumentUpdate
     /** @return The type of document this update is for. */
     const DocumentType& getType() const;
 
+    // Returns pointer to repo used for underlying update, or nullptr if default-constructed
+    const DocumentTypeRepo* getRepoPtr() const noexcept;
```

Comment:
This is small and simple enough to just keep in header file.
Consider moving getType() to header file too.

Reply:
Agreed, I just followed the existing source file convention 🙂

Reply:
`getType()` is not inlined because it explicitly downcasts to `DocumentType`, which is not included as part of the header file. So that explains that.

Reply:
K,